4788/src/main/include/Intake2.h

```diff
@@ -0,0 +1,65 @@
+#pragma once 
+
+#include "strategy/StrategySystem.h"
+#include "RobotMap.h"
+
+using actState = wml::actuators::BinaryActuatorState;
+
+enum class IntakeState {
+  STOWED, // Up, motor off
+  DEPLOYED, // Down, motor on
+  INTAKING, // Down, motor intake
+  EJECTING // Down, motor ejecting
+};
+
+class Intake : public wml::StrategySystem {
+ public:
+  Intake(wml::Gearbox &intakeGearbox,
+          wml::actuators::DoubleSolenoid &intakeActuator);
+
+  void SetIntake(const IntakeState st, double setpoint) {
+    _intakeState = st;
+    _intakeSetpoint = setpoint;
+  }
```

Comment:
The intake shouldn't need a setpoint, since stowed and deployed both don't move the motor, and intaking and ejecting run at a fixed speed.